src/main/java/qna/domain/Question.java

```diff
@@ -18,11 +19,10 @@
     @JoinColumn(foreignKey = @ForeignKey(name = "fk_question_writer"))
     private User writer;
 
-    @OneToMany(mappedBy = "question", cascade = CascadeType.ALL)
-    @Where(clause = "deleted = false")
-    @OrderBy("id ASC")
-    private List<Answer> answers = new ArrayList<>();
+    @Embedded
+    private Answers answers = new Answers();
```

Comment:
```suggestion
    @Embedded
    private Answers answers;
```

`생성자`에서 초기화 하는것이 좋겠지만 
현재는 `Answer`를 사용하는 아래에서 초기화 해줘도 좋겠네요 😄 

```
public void addAnswer(Answer answer) {
        this.answers = new Answers();
        answer.toQuestion(this);
        answers.add(answer);

```}